### PR TITLE
fix of bug #11876

### PR DESCRIPTION
**Pull request by pablolh**
https://github.com/mrdoob/three.js/issues/11876#issuecomment-322426016

### CODE REVIEW

File: src/audio/Audio.js [hunk 1]
```diff
@@ -102,11 +109,14 @@ Audio.prototype = Object.assign( Object.create( Object3D.prototype ), {
 			return;
 
 		}
-
-		this.source.stop();
-		this.startTime = this.context.currentTime;
-		this.isPlaying = false;
-
+		
+		if( this.isPlaying ) {
```
Comment (Mugen87) [inline src/audio/Audio.js]: just some code style issues. please write it like this:
```javascript
if ( this.isPlaying === true ){

   // code 

}
```

File: src/audio/Audio.js [hunk 2]
```diff
@@ -102,11 +109,14 @@ Audio.prototype = Object.assign( Object.create( Object3D.prototype ), {
...
+			this.source.stop();
+			this.offset += this.context.currentTime - this.startTime;
+			
+			this.isPlaying = false;
```
Comment (Mugen87) [inline src/audio/Audio.js]: one newline after this statement.

**Comment by Mugen87** [comment]
Um, why is this commit in your PR: f83fe541f6aac09e9a94943884f7d698dd55ba32

**Review by Mugen87 — CHANGES_REQUESTED** [review]

**Comment by donmccurdy** [comment]
@pablolh you may have gotten f83fe54 in here by starting from `master` branch instead of `dev`? I think `git revert f83fe54` would fix it...

**Comment by Mugen87** [comment]
@pablolh Maybe it's best to start with a fresh PR. As @donmccurdy said, checkout your feature branch from latest `dev` and then apply your changes. Before you submit the PR, you can validate if the list of commits is correct.

**Pull request closed without merging**